src/test/java/com/google/api/generator/engine/writer/ImportWriterVisitorTest.java

```diff
@@ -363,6 +364,41 @@ public void writeVariableExprImports_nestedReference() {
         writerVisitor.write());
   }
 
+  @Test
+  public void writeVariableExprImports_withAnnotations() {
+    Variable variable =
+        Variable.builder()
+            .setName("expr")
+            .setType(TypeNode.withReference(ConcreteReference.withClazz(Expr.class)))
+            .build();
+
+    TypeNode fakeAnnotationType =
+        TypeNode.withReference(
+            VaporReference.builder().setName("FakeAnnotation").setPakkage("com.foo.bar").build());
+
+    AnnotationNode annotation =
+        AnnotationNode.builder()
+            .setType(fakeAnnotationType)
+            .setDescription(
```

Comment:
Do we have to set descriptions? If not, removing it makes the test more concise.

Reply:
Ah true, not for the functionality this test intends to cover - replaced with Min's suggestion below!